error.js

```diff
@@ -1,3 +1,19 @@
+exports.customErrorHandler = (err, req, res, next) => {
+  console.log(err, "<-- err");
```

Comment:
If you want to keep a console.log in to see any errors, stick it in your 500 error handler. Then you know it's gone through all the error handlers and not been caught.